mirgecom/flux.py

```diff
@@ -72,7 +74,4 @@ def lfr_flux(cv_tpair, flux_func, normal, lam):
         object array of :class:`meshmode.dof_array.DOFArray` with the
         Lax-Friedrichs/Rusanov flux.
     """
-    flux_avg = 0.5*(flux_func(cv_tpair.int)
-                    + flux_func(cv_tpair.ext))
-    return flux_avg @ normal - 0.5*lam*(cv_tpair.ext
-                                        - cv_tpair.int)
+    return f_tpair.avg@normal - lam*jump(cv_tpair)/2
```

Comment:
`cv_tpair.diff` is a thing now, in case you want to use that

Reply:
(56e30d3)